src/apprt/gtk/Window.zig

```diff
@@ -125,7 +125,7 @@ pub fn init(self: *Window, app: *App) !void {
     self.notebook = Notebook.create(self);
 
     // If we are using Adwaita, then we can support the tab overview.
-    self.tab_overview = if ((comptime adwaita.versionAtLeast(1, 3, 0)) and adwaita.enabled(&self.app.config) and adwaita.versionAtLeast(1, 3, 0)) overview: {
+    self.tab_overview = if ((comptime adwaita.versionAtLeast(1, 4, 0)) and adwaita.enabled(&self.app.config) and adwaita.versionAtLeast(1, 3, 0)) overview: {
```

Comment:
There's still checks for two different versions of Adwaita - comptime 1.4.0 and runtime 1.3.0.